libraries/iio/iio.c

```diff
@@ -284,27 +293,6 @@ static ssize_t iio_phy_write(const char *buf, size_t len)
 	return -EINVAL;
 }
 
-/* Get string for channel id from channel type */
-static char *get_channel_id(enum iio_chan_type type)
```

Comment:
Function using get_channel_id should be updated in this commit

Reply:
of course, i did it in a different commit, will try to move changes around